documents/policies/external/Competition Requirements.md

```diff
@@ -56,36 +61,52 @@ The WCA Delegate must submit the following information when requesting approval
    2. The location should also include subregions of the country that are common to include with addresses (e.g. the state a U.S. city is located in).
    3. Competitions will be accepted regarding proximity if there are no other competitions within 10 km driving distance and less than 5 days away.
-      1. Otherwise, the WCA Competition Announcement Team should review the circumstances of the proposed competition. The WCA Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Possible arguments that support such a competition being accepted include:
+      1. Otherwise, the WCA Competition Announcement Team should review the circumstances of the proposed competition. The WCA Delegate requesting approval for such a competition must provide further information on why the competition should still be accepted. Reasons that will result in an approval include:
          1. The two competitions don’t have events in common.
          2. The two competitions take place in different countries.
-   4. 3x3x3 Fewest Moves simultaneous competitions
+   4. The WCA Competition Announcement Team, together with the Senior Delegate, may impose additional requirements for venues, such as which types of venues are allowed to be used within a region.
+      1. Competitions may not be held in a vehicle (e.g. ferry, cruise, plane) unless the vehicle has a dedicated room to hold the event.
+   5. 3x3x3 Fewest Moves simultaneous competitions
       1. 3x3x3 Fewest Moves simultaneous competitions are WCA competitions having only the 3x3x3 Fewest Moves event and taking place simultaneously in different locations.
-         1. Any location must be announced at least 28 days (four weeks) before 00:00 UTC of the start date of the competition on the WCA website.
-         2. If any location of such competitions is closer than 10 km to any other competition, then
+         1. Any location must be announced at least 14 days (two weeks) before 00:00 UTC of the start date of the competition on the WCA website.
+         2. If any location of such competitions is closer than 10 km to any other competition, then:
             1. both competitions should be at least 5 days apart, or
             2. the other competition must not have the 3x3x3 Fewest Moves event.
-      2. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
-         1. Time frames when both are held cannot overlap.
-   5. Series
+      2. Every location must clearly state who is the WCA Delegate for the venue.
+            1. There must be a listed non-Trainee Delegate at each venue.
+      3. A 3x3x3 Fewest Moves simultaneous competition cannot be part of any other regular competition.
+         1. Time frames when both are held cannot overlap. There must be at least one hour scheduled between the final scheduled event of one competition and the next scheduled event of the other.
```

Comment:
why do we need the time frames piece and the 1 hour restriction? To me the policy requirement was always about not being able to count the results of an FMC simultaneous competition towards two competitions (i.e. determining the podium of a local comp and the simultaneous competition). I see absolutely no issue with a location of an FMC simultaneous competition taking place in the side room of another local competition, for example.